ctapipe/calib/camera/tests/test_calibrator.py

```diff
@@ -177,6 +177,21 @@ def test_dl1_charge_calib(example_subarray):
         event.dl1.tel[telid].peak_time, mid / sampling_rate, atol=1
     )
 
+    # test deactivating timing corrections
+    calibrator.apply_waveform_time_shift = False
+    calibrator(event)
+    # times should still be correct, since peaks are shifted
+    np.testing.assert_allclose(
+        event.dl1.tel[telid].peak_time, mid / sampling_rate, atol=1
```

Comment:
Same result as with timing corrections activated - so not really a useful test

Reply:
Yes, but I don't have a good idea to test this.

Do you have a proposal? Maybe using the global peak integrator with large timeshifts?

Reply:
I now added tests using the `GlobalPeakWindowSum` that demonstrate nicely that the option works and why the peak time shifting is important in general.